mod/bookmarks/elgg-plugin.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+return [
+	'actions' => [
+		'logged_in' => [
+			'bookmarks/save' => 'bookmarks/save.php',
+			'bookmarks/delete' => 'bookmarks/delete.php',
+		],
+	],
+	'views' => [
+		'default' => [
+			'bookmarks/' => __DIR__ . '/graphics',
```

Comment:
We could define them as variables before requiring the file. Might nake sense to inject install path, root path, plgins path and root.